lib/generators/engine_cart/install_generator.rb

```diff
@@ -52,8 +52,7 @@ def add_gemfile_include
   # the below comes from engine_cart, a gem used to test this Rails engine gem in the context of a Rails app
   file = File.expand_path("Gemfile", ENV['ENGINE_CART_DESTINATION'] || ENV['RAILS_ROOT'] || File.expand_path("../spec/internal", __FILE__))
   if File.exist?(file)
-    puts "Loading \#{file} ..." if $DEBUG # `ruby -d` or `bundle -v`
-    instance_eval File.read(file)
+    eval_gemfile file
   else
     # we get here when we haven't yet generated the testing app via engine_cart
     gem 'rails', ENV['RAILS_VERSION'] if ENV['RAILS_VERSION']
```

Comment:
if this works, it would be a huge benefit, def one of the biggest problems I have with engine_cart. 

I wonder if the whole engine_cart-supplied text in the Gemfile should be encapsulated in a single:

      EngineCart.load_gemfile(__FILE__)

Or something like that. That would allow future versions of EngineCart to keep fine-tuning this functionality, and people to get the new functionality when they upgrade EngineCart without having to manually go and copy and paste it into their Gemfile.  As it is, this change, for instance, to get it people will have to both upgrade EC, and copy and paste (from where?) the new code into their Gemfile.